templates/jinja_macros/navbar.html

```diff
@@ -26,23 +26,19 @@
             <a href="/" class="navbar-item">Home</a>
             {% block navbar %}{% endblock %}
         </div>
-        <!--TODO move search bar down a little-->
+
         {% if session["logged_in"] %}
-            <div class="navbar-search has-background-grey-lighter">
-                <div class="field has-addons has-background-grey-lighter">
-                    <div class="control has-background-grey-lighter search">
-                        <input class="input has-background-grey-lighter has-text-black"
-                               id="search"
-                               placeholder="Search">
-                    </div>
-                    <div class="control" onclick="submit_search()">
-                        <input id="search"
-                               class="button has-background-grey-lighter has-text-black has-border-bottom-right-radius search_button"
-                               type="submit"
-                               value="Search">
-                    </div>
+        <!--TODO keep last search after search in search bar-->

```

Comment:
I think we should remove this comment and make a new issue for that UI fix instead.